schematools/cli.py

```diff
@@ -40,10 +46,33 @@
     help="Url where valid amsterdam schema files are found. "
     "SCHEMA_URL can also be provided as environment variable.",
 )
+
 argument_schema_location = click.argument(
     "schema_location", metavar="(DATASET-ID | DATASET-FILENAME)",
 )
 
+option_profile_url = click.option(
+    "--profile-url",
+    envvar="PROFILE_URL",
+    default=DEFAULT_PROFILE_URL,
+    show_default=True,
+    required=True,
+    help="Url where valid amsterdam profile files are found. "
+    "PROFILE_URL can also be provided as environment variable.",
+)
+
+argument_profile_location = click.argument(
+    "profile_location", metavar="(PROFILE-FILENAME | NONE)",
+)
+
+
+argument_role = click.argument(
+    "role",
+)
```

Comment:
als je `black` over je files draait krijg je waarschijnlijk een andere layout. Dit is nog goed om te doen (zelfs default in te stellen in je editor, dat `black` wordt gedraaid bij het opslaan)

Reply:
Ga ik doen!